docs/user/canvas.asciidoc

```diff
@@ -18,16 +18,170 @@ With Canvas, you can:
+
+[float]
+[[create-workpads]]
+== Create workpads
+
+A _workpad_ provides you with a workspace where you can build presentations of your live data.
+
+[float]
+[[start-with-a-blank-workpad]]
+=== Start with a blank workpad
+
+To use the background colors, images, and data of your choice, start with a blank workpad.
+
+. Open the menu, then go to *Canvas*.
+
+. On the *Canvas workpads* view, click *Create workpad*.
+
+. Add a *Name* to your workpad.
+
+. In the *Width* and *Height* fields, specify the size.
+
+. Select the layout.
++
+For example, click *720p* for a traditional presentation layout.
+
+. Click the *Background color* picker, then select the background color for your workpad.
++
+[role="screenshot"]
+image::images/canvas-background-color-picker.png[Canvas color picker]
+
+[float]
+[[create-workpads-from-templates]]
+=== Create workpads from templates
+
+If you're unsure about where to start, you can use one of the preconfigured templates that come with Canvas.
+
+. Open the menu, then go to *Canvas*.
+
+. On the *Canvas workpads* view, select *Templates*.
+
+. Click the preconfigured template that you want to use.
+
+. Add your own *Name* to the workpad.
+
+[float]
+[[import-existing-workpads]]
+=== Import existing workpads
+
+When you want to use a workpad that someone else has already started, import the JSON file into Canvas.
+
+. Open the menu, then go to *Canvas*.
+
+. On the *Canvas workpads* view, click and drag the file to the *Import workpad JSON file* field.
+
+[float]
+[[use-sample-data-workpads]]
+=== Use sample data workpads
+
+Each of the sample data sets comes with a Canvas workpad that you can use for your own workpad inspiration.
+
+. Add a {kibana-ref}/add-sample-data.html[sample data set].
+
+. On the *Add Data* page, click *View data*, then select *Canvas*.
+
+[float]
+[[add-canvas-elements]]
+== Add elements
+
+Create a story about your data by adding elements to your workpad that include images, text, charts, and more. You can create your own elements and connect them to your data sources, add saved objects, and add your own images.
+
+[float]
+[[create-elements]]
+=== Create elements
+
+Choose the type of element you want to use, then use the preconfigured demo data to familiarize yourself with the element. When you're ready, connect the element to your own data. By default, most of the elements you create use 
+demo data until you change the data source. The demo data includes a small data set that you can use to experiment with your element.
+
+To begin, click *Add element*, then select the element you want to use.
+
+[role="screenshot"]
+image::images/canvas-element-select.gif[Canvas elements]
+
+When you're ready to connect the element to your data, select *Data*, then select one of the following data sources:
+
+* *{es} SQL* &mdash; Access your data in {es} using SQL syntax. For information about SQL syntax, refer to {ref}/sql-spec.html[SQL language].
```

Comment:
How about removing the last sentence, then making "SQL syntax" in the first sentence the link.